graphql/image_test.go

```diff
@@ -65,6 +65,41 @@ func TestToolchains(t *testing.T) {
 	assert.Equal(t, testToolchain, utility.FromStringPtr(res[0].Name))
 }
 
+func TestEvents(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestEvents")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+
+	// Returns the correct number of events according to the limit.
+	imageID := "ubuntu2204"
+	image1 := model.APIImage{
+		ID: &imageID,
+	}
+	res, err := config.Resolvers.Image().Events(ctx, &image1, 5, 0)
+	require.NoError(t, err)
+	assert.Len(t, res, 5)
+
+	// Does not return the same events in different pages.
```

Comment:
if we want to test this, could we instead put the event.AMIAfter into a slice for  res and use utility.stringSliceContains on the res2 like this? 

https://github.com/bynn/evergreen/blob/ee98c0f5b0aa68d8af270f2d1c15474869b5694c/model/project_test.go#L1988-L1990